### PR TITLE
Check squash commit and converation

### PR DESCRIPTION
**Pull request by mohamed-imas**
pls make a request review for this with comment on any change

### CODE REVIEW

File: file7.txt [hunk 1]
```diff
@@ -1,2 +1,4 @@
 Tis
```
Comment (valeriy-fedotov) [inline file7.txt:1]: That would be good to change that line.

File: file5.txt [hunk 2]
```diff
@@ -1 +1,2 @@
 Unuseful file.
```
Comment (valeriy-fedotov) [inline file5.txt:1]: Common, it's very very useful!

**Review by valeriy-fedotov — CHANGES_REQUESTED** [review]

File: file7.txt (hunk 1 above)
Comment (mohamed-imas) [reply file7.txt:1]: will do next time

File: file5.txt (hunk 2 above)
Comment (mohamed-imas) [reply file5.txt:1]: in another life

**Comment by mohamed-imas** [comment]
we found out that  squashing commits and force push to the feature branch will **not** make following disappear 
resolved conversations/history of comments in the previous commit changes

**Pull request merged**